CHANGES.md

```diff
@@ -12,11 +12,32 @@ Tags:
 # 2.3.0 (Unreleased)
 
 ### Added
-- Source code rendering (@Julow, @panglesd, #909)
+- Source code rendering (@Julow, @panglesd, @jonludlam #909, #996, #993, #982)
 - Handle tables markup (@panglesd, @gpetiot, #893)
+- Initial support for assets (@trefis, #975)
+- odoc-parser remerged (@jonludlam, #973)
+  This includes table support (@gpetiot, @panglesd, ocaml-doc/odoc-parser#11
```

Comment:
Things related to tables should be moved to "Handle tables markup" above.

The odoc-parser remerge should be in the "Changed" section (here it's in "Added").

Reply:
Added seemed more appropriate to me, since we've added a whole opam package.